### PR TITLE
BorderColor, BorderWidth and Icon Background and Tint Color

### PR DESCRIPTION
**Pull request by MamtaKukreja**
Allow to customize border width and border color

### CODE REVIEW

File: TTGSnackbar/TTGSnackbar.swift [hunk 1]
```diff
@@ -116,6 +116,27 @@ open class TTGSnackbar: UIView {
         }
     }
     
+    /// Border Color of snackbar. Default is black
+    @objc open dynamic var borderColor: UIColor = .black {
```
Comment (zekunyan) [inline TTGSnackbar/TTGSnackbar.swift]: should be `@objc open dynamic var borderColor: UIColor? = .clear`

File: TTGSnackbar/TTGSnackbar.swift [hunk 2]
```diff
@@ -116,6 +116,27 @@ open class TTGSnackbar: UIView {
...
+        didSet {
+            layer.borderColor = borderColor.cgColor
```
Comment (zekunyan) [inline TTGSnackbar/TTGSnackbar.swift]: layer.borderColor = borderColor?.cgColor

File: TTGSnackbar/TTGSnackbar.swift [hunk 3]
```diff
@@ -116,6 +116,27 @@ open class TTGSnackbar: UIView {
...
+        }
+    }
+    
+    /// Border width of snackbar. Default is 1
+    @objc open dynamic var borderWidth: Int = 1 {
```
Comment (zekunyan) [inline TTGSnackbar/TTGSnackbar.swift]: `Int` is wrong. Should be `CGFloat`

File: TTGSnackbar/TTGSnackbar.swift [hunk 4]
```diff
@@ -263,6 +284,13 @@ open class TTGSnackbar: UIView {
         }
     }
     
+    /// Icon tint color
+    @objc open dynamic var iconTint: UIColor? = .black {
```
Comment (zekunyan) [inline TTGSnackbar/TTGSnackbar.swift]: Should be `@objc open dynamic var iconTintColor: UIColor! = .clear`

File: TTGSnackbar/TTGSnackbar.swift [hunk 5]
```diff
@@ -116,6 +116,27 @@ open class TTGSnackbar: UIView {
...
+    
+    /// Border width of snackbar. Default is 1
+    @objc open dynamic var borderWidth: Int = 1 {
+        didSet {
+            layer.borderWidth = borderWidth
+        }
+    }
+    
+    /// Image background color. Default is clear.
+    @objc open dynamic var imageBackgroundColor: UIColor = UIColor.clear {
```
Comment (zekunyan) [inline TTGSnackbar/TTGSnackbar.swift]: Should be `@objc open dynamic var iconBackgroundColor: UIColor?`

**Comment by MamtaKukreja** [comment]
@zekunyan I have made all changes as per comments. Please review again.

**Comment by MamtaKukreja** [comment]
@zekunyan please update the pod for the same

**Comment by MamtaKukreja** [comment]
@zekunyan unable to use pod version 1.9.1

**Comment by zekunyan** [comment]
@MamtaKukreja  Try again.

**Comment by MamtaKukreja** [comment]
@zekunyan Works now! Thanks!

**Pull request merged**